docs/transport/transport.md

```diff
@@ -0,0 +1,65 @@
+# gittuf Remote Transfer Protocol
+
+Last Modified: July 1, 2024
+
+## Introduction
+
+Alongside the `gittuf` binary, gittuf ships with a custom remote transfer
+protocol binary. We call this the **transport** binary, named
+`git-remote-gittuf`.
+
+It's an easy way to get started with using gittuf on your repository, as it
+takes care of the following common operations for you:
+
+- Creating RSL entries upon pushing your changes
+- Verifying changes you pull down
+
+As time goes on, the transport will perform other "best-practice" operations
+automatically in the background.
+
+This transport is intended to serve as a drop-in replacement for the standard
+Git transport. For more information on the technical details of how Git transfer
+protocols work, visit
+[the official Git documentation.](https://git-scm.com/book/en/v2/Git-Internals-Transfer-Protocols)
+
+## How to Obtain
+
+### `make install`
+
+Running `make install` will compile the transport and place it in your `GOBIN`,
+which is the same location that the `gittuf` binary is installed into. 
+
+### Downloading a release from GitHub
```

Comment:
We don't want to add this to releases just yet. Let's get it more stable.

Reply:
I'd add a clarifying note saying currently it must be installed from source. I'd also point to go install directly rather than make, as the latter needs them to clone the repo.